contents/javascript/Javascript-framework-Angular.md

```diff
@@ -0,0 +1,199 @@
+<frontmatter>
+  title: Introduction to Angular
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+{{ navbar | safe }}
+
+<div class="website-content">
+
+# Introduction to Angular
+
+Author(s): [Ronak Lakhotia](https://github.com/RonakLakhotia)
+
+## What Is Angular?
+
+Angular is one of the popular Javascript frameworks used for building web applications.
+The framework aims to make development and testing of enterprise web applications easier. Angular is maintained by Google, as well as a community of developers.
+
+>Angular started as a side project. Back in 2009, Miško Hevery and Adam Abrons released a project that would help developers, build web applications using simple HTML tags. The name “Angular” comes from the angle brackets, or < >, that surround all HTML tags.
+>
+>Miško described the idea behind the framework in an <a href="https://www.infoworld.com/article/2612801/javascript/whats-so-special-about-googles-angularjs.html">interview done in 2013</a>. 
+
+
+## Why learn Angular?
+
+Angular is a complex framework and it can be challenging for new developers to understand.
+But the advantages Angular provides justifies the steep learning curve.
+
+1. **Angular supports Single Page Applications (SPAs).**
+
+    <box>Single Page Applications are a type of web applications that load a single HTML page, and the page is updated dynamically according to the interaction of the user. SPAs can communicate with the back-end servers without refreshing the full web page, for loading data in the application. They provide a better user experience.</box>
```

Comment:
> SPAs can communicate with the back-end servers

Don't need `the`

> for loading data in the application.

This concept can be explained in another sentence, doesn't fit well in this sentence.